src/sentry/event_manager.py

```diff
@@ -523,6 +524,18 @@ def stringify(f):
         if data['transaction']:
             data['transaction'] = trim(data['transaction'], MAX_CULPRIT_LENGTH)
 
+        ip_address = get_path(data, ['sentry.interfaces.User', 'ip_address'])
+        if ip_address:
+            geo = geo_by_addr(ip_address)
+            if geo:
+                data['sentry.interfaces.User']['geo'] = {
```

Comment:
Is there a reason we want to actually store this on the payload vs just augmenting when needed? e.g. on read, apply the geo lookup.

Reply:
I did it in Snuba first but we decided to move it up so that it would be available on any Event (with an `ip_address`) in Kafka.

https://github.com/getsentry/snuba/pull/88